### PR TITLE
Enable build tags for Windows Releases

### PR DESCRIPTION
**Pull request by hmaarrfk**
@jni por-favor. 

Closes #https://github.com/scikit-image/scikit-image/issues/4270

### CODE REVIEW

**Comment by hmaarrfk** [comment]
Ahahhh

**Comment by jni** [comment]
@hmaarrfk please ping me again when this is de-messed 😂

**Comment by hmaarrfk** [comment]
Ok success!
```
Processing c:\projects\scikit-image-wheels\scikit-image\dist\scikit_image-0.16.2-1-cp36-cp36m-win32.whl
```

File: .appveyor.yml [hunk 1]
```diff
@@ -13,6 +13,11 @@ environment:
   NP_BUILD_DEP: "numpy==1.14.5"
   NP_TEST_DEP: "numpy==1.14.5"
   DAILY_COMMIT: master
+  # Unless there is a problem in releasing wheels, this should be set to an
+  # empty string. If there is a problem releasing wheels and a new build
+  # must be uploaded, then "--build-number <some number>" can be set
+  # https://github.com/MacPython/wiki/wiki/Build-Tags
+  BDIST_WHEEL_ARGS: "--build-number 1"
```
Comment (jni) [inline .appveyor.yml]: Nice paragraph! Could you forward-port to master when this is done, with the empty string? =)

Comment (hmaarrfk) [reply .appveyor.yml]: I couldn't figure out how to do it for linux/mac too... I would rather not forward port until somebody finds out how ???

Comment (jni) [reply .appveyor.yml]: I'm confused... Isn't the process exactly the same but on Travis?

Comment (hmaarrfk) [reply .appveyor.yml]: Not really, for Linux and Mac the build process is pretty obscure and jumps through many more scripts in muktibuild. Finally for Linux is gets spawned as port of an isolated environment in Dockers so you need to be careful in how you pass parameters to it.

**Comment by hmaarrfk** [comment]
Anyway, merging since for now, we only need the windows 3.6 and 3.7 builds.

**Comment by matthew-brett** [comment]
Just to say - I never tried to automate build tags - I just built the wheels the usual way, then downloaded them, added the build tags, and uploaded them to pypi.  I used something like the [add_build.py]https://gist.github.com/matthew-brett/1b2e76d0f4993c1ff6c1ed4d16a7384d) script to do that.

When I had to do this in the past, automating it didn't seem like it was worth the effort, given I only need to do it rarely, and doing it manually isn't much work.

**Pull request merged**